build/init.yml

```diff
@@ -16,10 +16,7 @@ steps:
     versionSpec: '5.6.0'
 
 - task: UseDotNet@2
-  displayName: 'Use .NET Core SDK 3.1.300'
+  displayName: 'Use .NET Core SDK 3.1.405'
   inputs:
     packageType: sdk
-    version: '3.1.300'
-
-- script: dotnet tool restore
-  displayName: 'Restore .NET tools'
+    version: '3.1.405'
```

Comment:
Is there a reason to update the .NET Core version? I looked at the history of the file and it looks like the version is updated when people happen to be changing something related.

Reply:
No, there was no specific reason to update it - just felt like it would be nice to use the latest version, especially since there have been many security patches since 3.1.300.